src/alpha-lab/blink_detection.md

```diff
@@ -0,0 +1,101 @@
+
+::: tip
+If you are interested in learning more about the technical details of the underlying blink detection pipeline, we invite you to also have a look at our [**white paper**](https://docs.google.com/document/d/1JLBhC7fmBr6BR59IT3cWgYyqiaM8HLpFxv5KImrN-qE/export?format=pdf).
+:::
+
+### Getting started
+
+To get started, check out the accompanying [**GitHub repository**](https://github.com/pupil-labs/real-time-blink-detection). There you can find detailed instructions on how to install all required packages as well as a [**Jupyter notebook**](https://github.com/pupil-labs/real-time-blink-detection/blob/main/blink_detection.ipynb) that contains all the code needed along with instructions on how to run everything.
+
+If you don't have a recording at hand, worry not! The GitHub repository comes with an example Neon recording that you can explore.
+
+### Results
+
+#### Part 1: Posthoc blink detection
+
+Once you have run the first part of the notebook, you will get an output from which you can derive a number of statistics that provide various insights about your recording: the total number of detected blinks, the estimated blink rate (in Hz), and the average blink duration (in seconds), as well as eyelid closing and re-opening durations (in seconds).
+
+<div class="pb-4" style="display:flex;justify-content:left;">
+  <v-img
+    class="rounded"
+    :src="require('../media/alpha-lab/eye_blinks_statistics.png')"
+    max-width=500px
+  >
+  </v-img>
+</div>
+
+<font size=2>**Figure 1.** Blink statistics extracted from the provided example Neon recording.</font>
+
+You will additionally get a visual representation of the detected blinks, providing an overview of their temporal distribution:
+
+
+![Visual blink representation](../media/alpha-lab/eye_blinks_timeline.png)
+<font size=2>**Figure 2.** Visual representation of the detected blinks in the example Neon recording. Green shaded areas depict detected blink events.</font>
+
+When examining blink durations and the eyelid closing and re-opening times, you may notice that the total blink duration doesn't consistently equate to the sum of the durations of the closing and re-opening phases. This discrepancy arises because there are some frames during the period between closing and reopening of the eyelids where there is negligible movement, and thus, they are not classified as part of either the opening or the closing phase. However, blink duration is calculated from the start time of the eyelid closing to the last frame identified as part of the re-opening sequence and will, thus, almost always be marginally longer than the sum of the individual events.<br>
```

Comment:
Can we simplify this paragraph? Lots of important info here and want to ensure a reader doesn't misunderstand. It is important conceptual underpinning for how the data is reported.

Currently it reads in the beginning as if it is a problem/bug; but really it's just that (if I understand correctly) there are 3 "states" eye closing, eye closed, eye opening and eye closed is omitted from eyelid closing and eyelid opening. Hence the mismatch between blink duration and closing and opening.